fontc_crater/README.md

```diff
@@ -22,14 +22,17 @@ script in the [`fontc_crater` repo][crater-repo] and results are posted to
 To run in CI mode locally to play with the html output:
 
 When run in CI, we want to ensure that we have complete control over the set of
-python pacakges we are comparing against; to ensure this we use `constraints.txt`
-in the root `resources/scripts` directory. To update the set of python packages
-in use, you need to update this file:
+python pacakges we are comparing against; to ensure this we use `pip-compile`
+(part of [`pip-tools`]) to generate a pinned `requirements.txt` from a
+`requirements.in` file in the root `resources/scripts` directory. To update the
+set of python packages in use, you need to update this file:
 
 - `rm -rf` your existing venv
 - create a new venv and activate it
-- `pip install -r resources/scripts/requirements.txt`
-- `pip freeze > resources/scripts/constraints.txt`
+- install pip-tools with `python -m pip install pip-tools`
+- delete `resources/scripts/requirements.txt`
```

Comment:
I don't think you need to delete it as it will be overwritten, but it's ok

Reply:
have you actually tried _not_ to delete the existing requirements.txt? I think pip-compile is smart enough to only update what's strictly needed. Unless you really want to achieve the equivalent of `pip-compile --upgrade requirements.in` (which tries to upgrade all deps to latest version, not only the ones that are required) with your manual removal? You may well use the pip-compile --upgrade flag then.

Reply:
if there's an existing requirements.txt it doesn't modify it, it just prints to stdout..

Reply:
Did you provide an output path -o requirements.txt? I think it prints to stout if you don't

Reply:
it creates `requirements.txt` if it doesn't exist, if there are no arguments.